### ISSUE TITLE
Sizes of tensors must match except in dimension 0

### ISSUE THREAD

**Issue by NOTlinnkoln** [TinyTerra/ComfyUI_tinyterraNodes#189]
![Image](https://github.com/user-attachments/assets/8feb552a-1763-4b27-a27d-5150e7dff4b9)

![Image](https://github.com/user-attachments/assets/728da569-f33c-48d2-b4d2-bd808d782ce9)

```
# ComfyUI Error Report
## Error Details
- **Node ID:** 11
- **Node Type:** ttN advPlot images
- **Exception Type:** RuntimeError
- **Exception Message:** Sizes of tensors must match except in dimension 0. Expected size 512 but got size 1024 for tensor number 2 in the list.
## Stack Trace
```
  File "D:\AI\ComfyUI_windows_portable__Build-v3-1_Playground\ComfyUI\execution.py", line 327, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "D:\AI\ComfyUI_windows_portable__Build-v3-1_Playground\ComfyUI\execution.py", line 202, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "D:\AI\ComfyUI_windows_portable__Build-v3-1_Playground\ComfyUI\execution.py", line 174, in _map_node_over_list
    process_inputs(input_dict, i)

  File "D:\AI\ComfyUI_windows_portable__Build-v3-1_Playground\ComfyUI\execution.py", line 163, in process_inputs
    results.append(getattr(obj, func)(**inputs))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "D:\AI\ComfyUI_windows_portable__Build-v3-1_Playground\ComfyUI\custom_nodes\ComfyUI_tinyterraNodes\ttNpy\tinyterraNodes.py", line 2666, in plot
    plot_image, images, samples = plotter.xy_plot_process()
                                  ^^^^^^^^^^^^^^^^^^^^^^^^^

  File "D:\AI\ComfyUI_windows_portable__Build-v3-1_Playground\ComfyUI\custom_nodes\ComfyUI_tinyterraNodes\ttNpy\tinyterraNodes.py", line 896, in xy_plot_process
    images_out = torch.cat(images, dim=0)
                 ^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by NOTlinnkoln** [comment]
feels like the problem is in the compositing node for plot. Because generations run just fine

**Comment by NOTlinnkoln** [comment]
Probably problem related with fixed aspect ratio of images in array
